papyri/tests/expected/numpy:linspace.expected

```diff
@@ -11,22 +11,20 @@ Returns num evenly spaced samples, calculated over the interval [`start`, stop
 
 The endpoint of the interval can optionally be excluded.
 
-╭──────────────────────────────────────────────────────────────────────────────╮
-│ {'type': 'admonitionTitle', 'children': [{'type': 'text', 'value': 'versionc │
-╰──────────────────────────────────────────────────────────────────────────────╯
-╭──────────────────────────────────────────────────────────────────────────────╮
-│ Non-scalar `start` and `stop` are now supported.                             │
+┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+┃ |?|: versionchanged 1.16.0                                                   ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
+│ Non-scalar start and stop are now supported.                                 │
```

Comment:
this missing `` ` `` in start and stop seem to be wrong.

Reply:
that is correct, it is just that they are resolved now, and show in color. Maybe we should have a way to have markup like `[link]`/`[/link]` when rendering with no colors.